src/SFA.DAS.AANHub.Application/Apprentices/Queries/GetApprenticeMemberQueryValidator.cs

```diff
@@ -1,14 +1,22 @@
 ﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
 
 namespace SFA.DAS.AANHub.Application.Apprentices.Queries
 {
     public class GetApprenticeMemberQueryValidator : AbstractValidator<GetApprenticeMemberQuery>
     {
-        public GetApprenticeMemberQueryValidator()
+        public const string ApprenticeIdNotFoundMessage = "ApprenticeId was not found";
+        public GetApprenticeMemberQueryValidator(IApprenticesReadRepository apprenticesReadRepository)
         {
             RuleFor(a => a.ApprenticeId)
                 .NotNull()
-                .NotEmpty();
+                .NotEmpty()
+                .MustAsync(async (apprenticeid, cancellation) =>
```

Comment:
```suggestion
                .MustAsync(async (apprenticeId, cancellation) =>
```
Variables like this should be (lower) camel case

Reply:
I'm fairly sure you can set up VS to raise things like these as warnings too.

Reply:
Updated variable case. Thanks. I will look for the VS setting/standard